sdk/textanalytics/azure-ai-textanalytics/tests/test_recognize_linked_entities.py

```diff
@@ -586,3 +586,19 @@ def test_string_index_type_not_fail_v3(self, client):
         # make sure that the addition of the string_index_type kwarg for v3.1-preview.1 doesn't
         # cause v3.0 calls to fail
         client.recognize_linked_entities(["please don't fail"])
+
+    # currently only have this as playback since the dev endpoint is unreliable
+    @pytest.mark.playback_test_only
```

Comment:
can we open an issue to remove playback marker when endpoint works reliably? (sorry for nit, I feel like these kinds of things are too easy to forget about)

Reply:
np, good point [here](https://github.com/Azure/azure-sdk-for-python/issues/13448) you go bb